_locales/de/messages.json

```diff
@@ -51,7 +51,7 @@
         "message": "keiner"
     },
     "optionsCustomHeaderDescription": {
-        "message": "<p>Sie können hier einen benutzerdefinierten Header eintragen.</p><p>Sie können eine beliebige Zeichenfolge und die folgenden speziellen Blöcke verwenden:</p><ul><li><code>{num-tabs}</code>: Anzahl von Tabs</li><li><code>\\r\\n</code>: new line</li></ul><p>Beispiel: <code>From Export Tabs URLs (# links: {num-tabs})</code></p>"
+        "message": "<p>Sie können hier einen benutzerdefinierten Header eintragen.</p><p>Sie können eine beliebige Zeichenfolge und die folgenden speziellen Blöcke verwenden:</p><ul><li><code>{num-tabs}</code>: Anzahl von Tabs</li><li><code>\\r\\n</code>: neue Teile</li></ul><p>Beispiel: <code>From Export Tabs URLs (# links: {num-tabs})</code></p>"

```

Comment:
a little typo: should rather be `neue Zeile`

Reply:
> should rather be `neue Zeile`
Correct. Seems it missed that one.

Reply:
@pikim would you be so kind to correct this typo, that your PR can be merged?

Reply:
Done